etcdctlv3/command/member_command.go

```diff
@@ -166,12 +168,33 @@ func memberListCommandFunc(cmd *cobra.Command, args []string) {
 		ExitWithError(ExitError, err)
 	}
 
-	// use https://github.com/olekukonko/tablewriter to print out a pretty table?
-	for _, m := range resp.Members {
-		if len(m.Name) == 0 {
-			fmt.Printf("%16x[unstarted]: peerURLs=%s\n", m.ID, strings.Join(m.PeerURLs, ","))
-		} else {
-			fmt.Printf("%16x: name=%s peerURLs=%s clientURLs=%s isLeader=%v\n", m.ID, m.Name, strings.Join(m.PeerURLs, ","), strings.Join(m.ClientURLs, ","), m.IsLeader)
+	outputType, _ := cmd.Flags().GetString("write-out")
```

Comment:
i do not feel it is necessary to make every utility commands to accept -w. member list might be an exception when people want to programmatically parse the result.